packages/form-data/src/form-data.js

```diff
@@ -312,3 +318,21 @@ const BlobFile = class File {
 const panic = (error) => {
   throw error;
 };
+
+/**
+ *
+ * @param {Element} element
+ * @returns {element is HTMLSelectElement}
+ */
+function isSelectElement(element) {
```

Comment:
these are only used once, might as well inline them?

Reply:
doing so loses out on some type checking `Property 'value' does not exist on type 'Element'.` :/

Reply:
TYPESCRIPT!!!!!!

Reply:
i felt a jsdoc to typescript migration was a little 🤏 outside of the scope of this PR haha